packages/eslint-plugin/src/rules/file-not-included.test.ts

```diff
@@ -0,0 +1,31 @@
+import { tester } from 'src/test-utils'
```

Comment:
is this import intended ?

Reply:
@astahmer Yeah...this is what we use in testing. Or did you mean something else?

Reply:
Oh...it didn't satisfy typescript